packages/spor-elm/src/Spor/LineTagIcon.elm

```diff
@@ -0,0 +1,329 @@
+module Spor.LineTagIcon exposing (..)
+
+{-| A component for displaying line tag icons
+
+
+## Config
+
+@docs init, withVariant, withSize, withAdditionalStyle, withColor, withDescription
+
+
+## Display
+
+@docs toHtml
+
+-}
+
+import Css exposing (Color, Style)
+import Css.Global
+import Html.Styled as Html exposing (Html)
+import Html.Styled.Attributes as Attributes
+import Spor.Common.Types exposing (Size(..), Variant(..))
+import Spor.Icon.Transportation as Transportation
+import Spor.Token.Color.Alias as Alias
+import Spor.Token.Color.Linjetag as Linjetag
+import Svg.Styled as Svg exposing (Svg)
+
+
+{-| A component for displaying line tag icons
+-}
+type LineTagIcon
+    = LineTagIcon Options
+
+
+type alias Options =
+    { variant : Variant
+    , size : Size
+    , additionalStyle : Style
+    , color : Maybe Color
+    , description : Maybe String
```

Comment:
Trenger LineTagIcon-et en description? 🤔

Reply:
Akkurat nå gjør jeg det sånn fordi jeg skiller på walk-typen her inne (men det må ikke være sånn). Problemet per nå er at for å teste endringer, må jeg kopiere all koden over i det repoet som skal bruke det. Derfor hadde det vært fint å få ut en første versjon, slik at jeg enklere kan teste komponenten(e) og dermed gjøre flere små, iterative endringer (ettersom det er jeg som skal implementere bruken av dem også) 👍